supabase/migrations/20260711100000_updates_cache_invalidation.sql

```
@@ -0,0 +1,163 @@
+-- Targeted invalidation for the /updates colo cache
+-- (see supabase/functions/_backend/utils/updates_colo_cache.ts).
+--
+-- Whenever a row that feeds the update hot path changes, notify the
+-- triggers/cache_invalidate function (via pg_net, async, ~ms overhead per
+-- write) which fans the per-app token bump out to every regional plugin
+-- worker. The cache TTL remains the backstop if a call is lost.
+
+-- Notify helper: dedupes, caps and chunks the app list so the payload is
+-- always bounded (the fan-out endpoint enforces 100 ids per request; an
+-- org bigger than the cap falls back to the cache TTL for the tail).
+CREATE OR REPLACE FUNCTION public.notify_updates_cache_invalidation(p_app_ids text[])
```

SQLFluff (4.2.2)

[error] Line is too long (85 > 80).

(LT05)